spec/api-shell-spec.js

```diff
@@ -76,7 +77,8 @@ describe('shell module', () => {
         args: 'args2',
         appUserModelId: 'appUserModelId2',
         icon: 'icon2',
-        iconIndex: 2
+        iconIndex: 2,
+        toastActivatorClsid: '{C51A3996-CAD9-4934-848B-16285D4A1496}'
```

Comment:
this doesn't actually check that this option had any effect—if the code broke, this test would still pass. Can we add a check? Perhaps using `ReadShortcutLink`?

Reply:
It actually does. The 2 lines below write the change to disk and then read it back with a deep equal check.